pywbemcli/_common.py

```diff
@@ -239,15 +254,35 @@ def objects_sort(objects):
 
 def parse_wbemuri_str(wbemuri_str, namespace=None):
     """
-    Parse a string that is a wbemuri into a CIMInstanceName object.
+    Parse a string that is a wbemuri into a CIMInstanceName object.  This method
+    parses a string consistent with a wbemuri into a CIMInstanceName object.
+
+    If the wbem_uri includes a namespace that is parsed also.  If both
+    the namespace optional parameter exists and the wbemuri_str parameter
+    includes a namespace (and they are not the same), an exception is returned.
+
+    Returns:
+        CIMInstanceName instance object
+
+    Raises:
+        ClickException: if the input wbemuri_str is an invalid wbemuri.
+
     """
-    if namespace:
-        wbemuri_str = '/%s:%s' % (namespace, wbemuri_str)
+    # TODO remove this code when we resolve issue with pywbem issue #1359
+    # Issue is that 0.13.0 does not allow the form classname.keybindings
+    # without the : before the classname
+    if wbemuri_str and wbemuri_str[0] != ":":
```

Comment:
Issue pywbem/pywbem#1359 has been merged into master, so this circumvention code can be removed.

Reply:
I have to correct myself. That issue is your issue that you opened on this, and what was merged was a change on to_wbem_uri. Let's investigate this as part of that issue.